var/spack/repos/builtin/packages/cloc/package.py

```diff
@@ -11,8 +11,10 @@ class Cloc(Package):
     given files (may be archives such as compressed tarballs or zip files)
     and/or recursively below the given directories."""
     homepage = "https://github.com/AlDanial/cloc/"
-    url      = "https://github.com/AlDanial/cloc/releases/download/1.74/cloc-1.74.tar.gz"
+    url      = "https://github.com/AlDanial/cloc/archive/v1.90.tar.gz"
```

Comment:
Why not using the tarball that the maintainers uploaded:
```
https://github.com/AlDanial/cloc/releases/download/v1.90/cloc-1.90.tar.gz
```
? If we change to Github autogenerated tarballs we have to change all previous sha256 and we are more exposed to changes in the sea in the future (e.g. if the way archive gets generated changes).

Reply:
@alecbcs